pkg/scalers/liiklus/LiiklusService_grpc.pb.go

```diff
@@ -1,17 +1,17 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
 // - protoc-gen-go-grpc v1.3.0
-// - protoc             v4.23.2
```

Comment:
These changes are not related to your scaler too, I think.

Reply:
yes.